src/Discord.Net.Core/Entities/Users/IConnection.cs

```diff
@@ -1,14 +1,29 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 
 namespace Discord
 {
     public interface IConnection
     {
+        /// <summary>
+        ///     ID of the connection account.
+        /// </summary>
```

Comment:
All `summary` tags should begin with a verb. In this case, a getter field should begin with `Gets...`. This applies to every `get` fields proposed in this PR.